aws_lambda_powertools/utilities/data_classes/shared_functions.py

```diff
@@ -103,7 +137,7 @@ def get_multi_value_query_string_values(
 
     Returns
     -------
-    List[str]. optional
+    list[str]. optional
```

Comment:
Can we revert al the `List` --> `list` and `Dict` --> `dict` changes in this file? They are unrelated to the PR and when we do it we should do it project wide.

Reply:
Ok